src/checks/noDependencyProblems/noDependenciesWithoutTypes.ts

```diff
@@ -74,7 +74,7 @@ export default async function noDependenciesWithoutTypes() {
         // Show dependency name if Algolia's `index.getObject()` throws with an
         // error message (such as the error message "ObjectID does not exist"
         // when a package cannot be found in the index)
-        throw new Error(`Algolia error for \`${dependency}\`: ${err}`);
+        throw new Error(`Algolia error for \`${dependency}\`: ${err.message}`);
```

Comment:
- [ ] `err` needs TS casting to `Error` (in parentheses)
- [ ] let's rename it to `error`

Reply:
According to https://typescript.tv/best-practices/error-ts1196-catch-clause-variable-type-annotation/ I choose to add type `any` because typescript wouldnt let me pass the message property with `unknown`

Reply:
Since I am not 100% satisfied with `any` I researched for another solution.

This would be an alternative based on https://stackoverflow.com/a/70832746
```
catch (err) {
        // Show dependency name if Algolia's `index.getObject()` throws with an
        // error message (such as the error message "ObjectID does not exist"
        // when a package cannot be found in the index)
        const error = err as { message: string };
        throw new Error(
          `Algolia error for \`${dependency}\`: ${error.message}`,
        );
      }
```